### PR TITLE
Update README with a more clear note on wiring the ESP8266

### PR DESCRIPTION
**Pull request by janjongboom**
Copied from https://github.com/ARMmbed/connected-lights/blob/master/docs/4_connectivity.md#esp8266-wi-fi-module. It would be good for one of our tech writers to go over this README anyway.

cc @AnotherButler

### CODE REVIEW

**Comment by AnotherButler** [comment]
Do you want me to go ahead and edit this README, or shall I continue to wait for these changes to be made first?

**Comment by bulislaw** [comment]
The link does work now, and I agree it makes sense linking it. 
These passages are removed:

> ESP module needs to be connected to RX and TX UART pins (+ power and ground) on your target > board. That can be achieved using Grove shield or connected directly using jumper wires, please > note that not all Arduino form factor boards have UART compatible with the Grove shiled.

> For Grove shield TX has to be connected to D1 and RX to D0.

> Make sure that UART module you're connecting ESP to is different than the debug UART connected to your USB port.

I think they provide some general info that's good to have, that does not mean it cannot be improved, like connecting to you UART pins which are different than you USB UART and if you have grove shield they will be D1/D0.

And that's an useful addition:
> +**Note on NUCLEO boards:** On the NUCLEO boards pins `D0` and `D1` are used for serial communication with the computer. Use pins `D8` (to ESP8266 TX) and `D2` (to ESP8266 RX) instead.

**Comment by 0xc0170** [comment]
Whats the status of this one? @janjongboom

**Comment by janjongboom** [comment]
@AnotherButler can you handle this?

**Comment by AnotherButler** [comment]
@janjongboom Absolutely. I was waiting for the contents to be approved before editing. Is it ready for me to edit?

File: README.md [hunk 1]
```diff
@@ -19,11 +19,9 @@ ESP2866 is a fallback option and will be used if the build is for unsupported pl
 
 #### Connecting the ESP2866 ####
 
-ESP module needs to be connected to RX and TX UART pins (+ power and ground) on your target board. That can be achieved using Grove shield or connected directly using jumper wires, please note that not all Arduino form factor boards have UART compatible with the Grove shiled.
+To wire the ESP8266 module up to your development board, look at the [ESP8266 Cookbook page](https://developer.mbed.org/users/4180_1/notebook/using-the-esp8266-with-the-mbed-lpc1768/). In general this means hooking up the ESP8266 TX pin up to `D0` and the ESP8266 RX pin up to `D1` on your development board.
 
-For Grove shield TX has to be connected to D1 and RX to D0.
-
-Make sure that UART module you're connecting ESP to is different than the debug UART connected to your USB port.
+**Note on NUCLEO boards:** On the NUCLEO boards pins `D0` and `D1` are used for serial communication with the computer. Use pins `D8` (to ESP8266 TX) and `D2` (to ESP8266 RX) instead.
```
Comment (AnotherButler) [inline README.md]: Please add comma after "boards"

File: README.md [hunk 2]
```diff
@@ -19,11 +19,9 @@ ESP2866 is a fallback option and will be used if the build is for unsupported pl
 
 #### Connecting the ESP2866 ####
 
-ESP module needs to be connected to RX and TX UART pins (+ power and ground) on your target board. That can be achieved using Grove shield or connected directly using jumper wires, please note that not all Arduino form factor boards have UART compatible with the Grove shiled.
+To wire the ESP8266 module up to your development board, look at the [ESP8266 Cookbook page](https://developer.mbed.org/users/4180_1/notebook/using-the-esp8266-with-the-mbed-lpc1768/). In general this means hooking up the ESP8266 TX pin up to `D0` and the ESP8266 RX pin up to `D1` on your development board.
```
Comment (AnotherButler) [inline README.md]: Please change this sentence to the following for precise language and consistent comma use:

To connect the ESP8266 module to your development board, look at the [ESP8266 Cookbook page](https://developer.mbed.org/users/4180_1/notebook/using-the-esp8266-with-the-mbed-lpc1768/). In general, this means hooking up the ESP8266 TX pin to `D0` and the ESP8266 RX pin to `D1` on your development board.

**Comment by AnotherButler** [comment]
@janjongboom I don't have access to make these changes myself. Can you make them and merge?

**Comment by AnotherButler** [comment]
@janjongboom bump

**Comment by janjongboom** [comment]
@bulislaw The information in the sentence you mentioned is wrong. It does not tell anything about RESET pins being low / high, CH_PD needing to be connected to 3.3V. That information is in the cookbook page though, so I'd suggest keeping it there. 

I merged Amanda's changes, but you'll need to sign off on the actual merge.

**Comment by bulislaw** [comment]
@AnotherButler are you happy with the current state?

**Comment by AnotherButler** [comment]
@bulislaw Yes, this looks great. 👍

**Pull request merged**